hooks/post-checkout

```diff
@@ -29,7 +29,12 @@ function merge() {
 
   git fetch -v origin "${target_branch}"
   git checkout FETCH_HEAD
-  GIT_COMMITTER_NAME="github-merged-pr-buildkite-plugin" git merge "${BUILDKITE_COMMIT}"
+  # env vars to ensure merge is non-interactive
+  export GIT_MERGE_AUTOEDIT="no"
+  export GIT_COMMITTER_EMAIL="foo@bar"
```

Comment:
These will change git settings for subsequent steps, e.g. if people commit from their build. Any chance it can be for the single line?

Reply:
Yeah sure, I didn't do it initially because we'd end up with a long line but I was also worried about exports persisting...